.github/workflows/linux.yml

```diff
@@ -850,8 +850,6 @@ jobs:
 
       - name: TensorFlow Hub Tests - TF FE
         run: |
-          python3 -m pip install openvino --find-links=${{ env.INSTALL_DIR }}/tools
-
```

Comment:
why OpenVINO installation is removed?

Reply:
this is redundant step. We have the same step above